client/templates/typescript-fetch-templates/.npmignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
why is all this subdirectory `typescript-fertch-templates` inside the repo? is it copy of files from openapi-generator? are there any differences?

Reply:
There are some differences regarding error handling. Please see below.

https://github.com/symbol/statistics-service/pull/107#discussion_r750385359
![image](https://user-images.githubusercontent.com/6256269/144051865-85820074-52eb-4e23-aaa2-d825246541bc.png)

Reply:
in this case shouldn't it land in some separate repo (i.e. to be able to track changes/divergences from original)

Reply:
It looks like extracting templates(there is a command for this in the cli tool) and modifying them is a recommended way of introducing small changes to the generated code. Please see https://openapi-generator.tech/docs/templating

Reply:
Since we are only extracting the `runtime.ts` template and updating it using the recommended approach,  I think it's okay to have it embedded since it's not related to the core part of the generator it's less likely to be updated.

For the alternative approach that you mentioned, forking the repo and keeping the changes there sounds reasonable to me but not sure it'll pay off since we would need to fork the https://github.com/OpenAPITools/openapi-generator instead of https://github.com/OpenAPITools/openapi-generator-cli since the `openapi-generator-cli` is a node package wrapper of the java package `openapi-generator` and need to add the jar file built & packaged from the forked repository to the dependencies and instruct `openapi-generator-cli` to use the local jar dependency.